cloud-azure/src/main/java/com/sequenceiq/cloudbreak/cloud/azure/image/AzureImageSetupService.java

```diff
@@ -92,22 +92,15 @@ public ImageStatusResult checkImageStatus(AuthenticatedContext ac, CloudStack st
     private ImageStatusResult getImageStatusResult(String imageResourceGroupName, AzureClient client, AzureImageInfo azureImageInfo, String imageStorageName) {
         CopyState copyState = client.getCopyStatus(imageResourceGroupName, imageStorageName, IMAGES_CONTAINER, azureImageInfo.getImageName());
         boolean storageContainsImage = storageContainsImage(client, imageResourceGroupName, imageStorageName, azureImageInfo.getImageName());
-        if (copyState == null && storageContainsImage) {
+        if ((copyState.getCopyStatusType() == null || CopyStatusType.SUCCESS.equals(copyState.getCopyStatusType())) && storageContainsImage) {
```

Comment:
nullcheck for copyState removed, was this intentional?

Reply:
copyState can't be null now. By the way the CopyState previously was a class in the azure lib. This was deleted and I created a new CopyState which cannot be null. The copy status type CAN be null, this is the same when the CopyState was null.

Reply:
ok, fair enough, hopefully noone can break the code and make it null